training_config/dialog_qa.jsonnet

```diff
@@ -46,7 +46,7 @@
             "type": "gru",
             "bidirectional": true,
             "hidden_size": 100,
-            "input_size": 1124,
+            "input_size": 1144,
```

Comment:
is it worth putting a comment in the config file explaining why this has to be what it is?

(or even using jsonnet local variables to enforce the constraint?)

Reply:
Sorry for submitting a PR with errors. :disappointed_relieved:
But I still don't understand why this is 1144 instead of 1124, so I think it is helpful for us to put a comment explaining the reason for this setting.